api/orders/index.js

```diff
@@ -64,6 +64,12 @@
             }
             console.log("Received response: " + JSON.stringify(body));
             var jsonBody = JSON.parse(body);
+
+            if (jsonBody._links == undefined) {
+              callback(error);
```

Comment:
I don't think error will have anything since it's suppose to be caught by the first if, no?